content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+  logical place. That next logical place might just be the H1 of the dialog.
+</Caption>
+
+### Closing
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/40274682/228390156-dab0b413-3b90-4736-a0e7-8d5d30665c43.mp4"
+    />
+    <Caption>
+      Pressing esc, or selecting “cancel” should close the current dialog and return focus to the previous trigger. It
+      should not retain or submit any user input.
+    </Caption>
+  </div>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/40274682/228390297-d6d59818-1d1e-47bb-ac9d-cd1eccc57ba8.mp4"
+    />
+    <Caption>Selecting the submit button should submit any user input.</Caption>
+  </div>
+</Box>
+
+### Responsive
+
+Select panel can become a full screen dialog or a bottom sheet at narrow viewports. Support at 320px is required if the Select panel is larger at wider viewports. See [narrow-viewport placement](https://primer.style/design/components/dialog#narrow-viewport-placement) in the Dialog guidelines for more details.
+
+## Keyboard navigation
+
+### Trigger
+
+| Key                                                                                                                                                       | Description            |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Enter</Box> | Opens the Select panel. |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Space</Box> | Opens the Select panel. |
+
+### Select panel
+
+| Key                                                                                                                                                                                          | Description                                                                                                                 |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Enter</Box>                                    | Submits the information in the form when focused in the listbox. Activates buttons or links focused within the Select panel. |
```

Comment:
It might be good to call out what `Enter` should do when the Search Input is focussed, even if that is "it should do nothing", otherwise it could be left to interpretation.